web-admin/src/features/authentication/AvatarButton.svelte

```diff
@@ -32,64 +33,45 @@
   }
 
   const isDev = process.env.NODE_ENV === "development";
-
-  // Position the Menu popover
-  const [popperRef1, popperContent1] = createPopperActions();
-  const popperOptions1 = {
-    placement: "bottom-end",
-    strategy: "fixed",
-    modifiers: [{ name: "offset", options: { offset: [0, 4] } }],
-  };
-
-  // Position the View As User popover
-  const [popperRef2, popperContent2] = createPopperActions();
 </script>
 
-<Popover class="relative" let:close={close1}>
-  <PopoverButton use={[popperRef1]}>
+<DropdownMenu>
+  <DropdownMenuTrigger>
     <img
       src={$user.data?.user?.photoUrl}
       alt="avatar"
```

Comment:
Unrelated to this PR, really, but do we need a fallback here or are we confident that `photoUrl` will always be defined?

Reply:
It does look like this could be protected better, though we are [always getting a string in response](https://vscode.dev/github/rilldata/rill/blob/main/admin/server/auth/handlers.go#L171).